lib/features/popup-menu/PopupMenuItem.js

```diff
@@ -33,10 +33,14 @@ export default function PopupMenuItem(props) {
       class=${ classNames('entry', { selected }) }
       data-id=${ entry.id }
       title=${ entry.title || entry.label }
+      tabIndex="0"
       onClick=${ onAction }
+      onFocus=${ onMouseEnter }
+      onBlur=${ onMouseLeave }
       onMouseEnter=${ onMouseEnter }
       onMouseLeave=${ onMouseLeave }
       onDragStart=${ (event) => onAction(event, entry, 'dragstart') }
+      aria-role="button"
```

Comment:
If the items serve as buttons now, why don't we make them actual buttons? Spans make sense if we want to keep the select-option-like behavior but we decided against this as I understand.